### ISSUE TITLE
icons_and_splashscreens.js no longer needed with new "ionic resources" CLI feature?

### ISSUE THREAD

**Issue by djett** [diegonetto/generator-ionic#148]
The icons_and_splashscreens.js was an awesome addition to the project, however, now with ionics new resources feature in the Ionic CLI, there is no need for this.  

The Ionic CLI requires a single icon.png and splash.png in the resources directory.  Running "ionic resources" will generate all the icons and splash screens for every platform installed and place them in a folder structure within the resources directory.

Using the CLI feature while also having the hook, will mean that you will have duplicate icons and splash screen images in the resources folder.

**Comment by jimthedev** [comment]
I agree that this is no longer needed if you're using the ionic-cli. With that said, the current state of this project assumes that you're not using the ionic cli in favor of the grunt. Seems like this would be a good item to put into the ionic 1.0.0 backlog that is being considered [here](https://github.com/diegonetto/generator-ionic/issues/153)
